samtools.wdl

```diff
@@ -286,4 +346,20 @@ task View {
         memory: memory
         docker: dockerImage
     }
+
+    parameter_meta {
+        inFile: {description: "A BAM, SAM or CRAM file.", category: "required"}
```

Comment:
What about the [Sequence (Protein) Alignment Mapping](https://www.youtube.com/watch?v=mBcY3W5WgNU) files?